### PR TITLE
[16][REF] split web_view_leaflet_map into web_view_leaflet_map and web_leaflet_lib

### PR DESCRIPTION
**Pull request by legalsylvain**
To be used by other modules, like 'web_widget_map'. (see : https://github.com/OCA/web/pull/2953/)

### CODE REVIEW

**Review by yvaucher — APPROVED** [review]

File: web_view_leaflet_map/__manifest__.py [hunk 1]
```diff
@@ -4,8 +4,7 @@
 
 {
     "name": "Leaflet Map View (OpenStreetMap)",
-    "summary": "Integrate leaflet.js librairy with odoo"
-    " and add new 'leaflet_map' view, to display markers.",
+    "summary": "Add new 'leaflet_map' view, to display markers.",
     "version": "16.0.1.0.0",
```
Comment (yvaucher) [inline web_view_leaflet_map/__manifest__.py]: Version to raise?

Comment (legalsylvain) [reply web_view_leaflet_map/__manifest__.py]: > Version to raise?


Sorry. I don't understand.

**Review by florian-dacosta — APPROVED** [review]
Functional review

File: web_view_leaflet_map/__manifest__.py (hunk 1 above)
Comment (legalsylvain) [reply web_view_leaflet_map/__manifest__.py]: I get it ! Done. 

Can be merged now, @yvaucher . Thanks for your review !

**Comment by legalsylvain** [comment]
@yvaucher could you "ocabot merge nobump" here ? 

thanks !

**Comment by yvaucher** [comment]
/ocabot merge nobump


Here we go.

**Pull request merged**